ext/pdo_mysql/tests/mysql_pdo_test.inc

```diff
@@ -166,11 +166,11 @@ class MySQLPDOTest extends PDOTest {
                 preg_match('/Client API version.*mysqlnd/', $tmp));
     }
 
-    static function dropTestTable($db = NULL) {
+    static function dropTestTable($db = NULL, $tableName = 'test') {
```

Comment:
Maybe it makes sense to flip the param other in a follow up PR.

(Also adding types)

Reply:
Good idea, I'll have a look!

About adding types, in this PR or in a follow-up alos?

Reply:
Follow up PR :)

Reply:
Alright I'll have a look at it!